gitops/prer/create_gitops_prs.go

```diff
@@ -100,6 +103,40 @@ func bazelQuery(query string) *analysis.CqueryResult {
 	return qr
 }
 
+func getContext(workdir *git.Repo, branchName string) map[string]interface{} {
```

Comment:
```suggestion
func getStampingValues(workdir *git.Repo, branchName string) map[string]interface{} {
```
`getContext` is too generic.